app/src/pages/inside/projectSettingsPageContainer/content/testLibrary/environments/createEnvironmentModal/createEnvironmentModal.jsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reduxForm } from 'redux-form';
+import classNames from 'classnames/bind';
+import { useIntl } from 'react-intl';
+
+import { Modal } from '@reportportal/ui-kit';
+import { ExternalLink } from 'pages/inside/common/externalLink';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { withModal } from 'components/main/modal';
+import { hideModalAction } from 'controllers/modal';
+import { FieldProvider } from 'components/fields';
+import { FieldText } from 'componentLibrary/fieldText';
+import { FieldTextFlex } from 'componentLibrary/fieldTextFlex';
```

Comment:
Please use the `FieldText` and `FieldTextFlex` from the ui-kit.
Local copies will be removed once all existing usages will be migrated to the ui-kit version.